crates/core/src/client/mod.rs

```diff
@@ -525,6 +570,21 @@ impl<P: Provider + Send + Sync> KakarotStarknetApi<P> for KakarotClient<P> {
         &self.starknet_provider
     }
 
+    /// Returns the Starknet block number for a given block id.
+    async fn get_block_number(&self, block_id: &StarknetBlockId) -> Result<u64, EthApiError<P::Error>> {
```

Comment:
no "get"

Reply:
changed to `map_block_id_to_block_number`